go/kbfs/libfuse/mounter.go

```diff
@@ -92,28 +93,36 @@ func fuseMountDir(dir string, platformParams PlatformParams) (*fuse.Conn, error)
 	return c, nil
 }
 
-func isFusermountMountNotFoundError(err error) bool {
+func isFusermountMountNotFoundError(output []byte, err error) bool {
 	if err == nil {
 		return false
 	}
-	return strings.Contains(err.Error(), "not found in /etc/mtab")
+	return bytes.Contains(output, []byte("not found in /etc/mtab"))
 }
 
 func (m *mounter) Unmount() (err error) {
+	m.log.Info("Unmounting.")
 	dir := m.options.MountPoint
 	// Try normal unmount
 	switch runtime.GOOS {
 	case "darwin":
 		_, err = exec.Command("/sbin/umount", dir).Output()
 	case "linux":
-		_, fusermountErr := exec.Command("fusermount", "-u", dir).Output()
+		fusermountOutput, fusermountErr := exec.Command("fusermount", "-u", dir).CombinedOutput()
 		// Only clean up mountdir on a clean unmount.
 		if fusermountErr == nil {
+			m.log.Info("Successfully unmounted.")
```

Comment:
We don't end log messages or errors with punctuation in KBFS (here and two places below).

Reply:
fixed. So the rule is no punctuation for message and errors, but yes punctuation for comments? Is there a rule about capitalization as well?